From b431b9951639e6b2e7b91cdde47a18b96f0a9319 Mon Sep 17 00:00:00 2001
From: Anton Bambura <jenneron@protonmail.com>
Date: Mon, 14 Mar 2022 21:49:18 +0200
Subject: [PATCH 21/33] HACK: ARM: dts: exynos: peach-pi: disable HDMI audio

When enabling Internal Microphone, HDMI codec results in:
[  730.054945] hdmi-audio-codec hdmi-audio-codec.14.auto: Only one simultaneous stream supported!
[  730.054961] hdmi-audio-codec hdmi-audio-codec.14.auto: ASoC: error at snd_soc_dai_startup on i2s-hifi: -22
[  730.054978]  Primary: soc_pcm_open() failed (-22)

Signed-off-by: Anton Bambura <jenneron@protonmail.com>
Tested-by: Valentine Iourine <iourine@iourine.msk.su>
---
 arch/arm/boot/dts/exynos5800-peach-pi.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/exynos5800-peach-pi.dts b/arch/arm/boot/dts/exynos5800-peach-pi.dts
index e528a677026c..73b2616bd095 100644
--- a/arch/arm/boot/dts/exynos5800-peach-pi.dts
+++ b/arch/arm/boot/dts/exynos5800-peach-pi.dts
@@ -99,7 +99,7 @@ cpu {
 		};
 
 		codec {
-			sound-dai = <&max98091>, <&hdmi>;
+			sound-dai = <&max98091>;
 		};
 	};
 
-- 
2.38.0

